app/models/accounting/transaction.rb

```diff
@@ -37,14 +37,16 @@
 class Accounting::Transaction < ActiveRecord::Base
   QB_TRANSACTION_TYPES = %w(JournalEntry Deposit Purchase).freeze
   LOAN_TRANSACTION_TYPES = %i(disbursement)
+  LOAN_INTEREST_TYPE = 'interest'
 
   belongs_to :account, inverse_of: :transactions, foreign_key: :accounting_account_id
   belongs_to :project, inverse_of: :transactions, foreign_key: :project_id
   belongs_to :currency
 
   before_save :update_fields_from_quickbooks_data
 
-  validates :loan_transaction_type, :txn_date, :amount, :accounting_account_id, presence: true
+  validates :loan_transaction_type, :txn_date, :accounting_account_id, presence: true
+  validates :amount, presence: true, unless: -> { qb_id.blank? && qb_transaction_type == LOAN_INTEREST_TYPE }
```

Comment:
I could change it if you prefer, but the specs I wrote are to test this validation specifically. So I know it does work.